app/views/admins/services/show.html.haml

```diff
@@ -24,7 +24,7 @@
             %i.icon-chevron-down
       %tbody
         - @service_requests.each do |service_request|
-          -if current_admin.id ==  service_request.public_servant_id
+          -if current_admin.id ==  service_request.public_servant_id || current_admin.is_service_admin
```

Comment:
The - symbol should have one space separating it from code<br>Line is too long. [102/80]